src/components/Proposals/Description.js

```diff
@@ -108,6 +115,7 @@ function DescriptionStep({ step }) {
 /* eslint-enable react/prop-types */
 
 Description.propTypes = {
+  disableBadgeInteraction: PropTypes.bool,
```

Comment:
Should we put false as default in this prop?

Reply:
Good point, though it's falsy by default, being explicit wouldn't hurt.